lib/_tls_wrap.js

```diff
@@ -1051,7 +1051,7 @@ Server.prototype.setOptions = util.deprecate(function(options) {
 // SNI Contexts High-Level API
 Server.prototype.addContext = function(servername, context) {
   if (!servername) {
-    throw new ERR_TLS_REQUIRED_SERVER_NAME();
+    throw new ERR_MISSING_ARGS('servername');
```

Comment:
I think the convention here would be:
```
if (typeof servername !== 'string')
  throw new ERR_INVALID_ARG_TYPE('servername', 'string', arg);
```
to test both the presence of the arg, but also its type. If it's not a string it will throw anyway now just below on the call to `.replace()`, but throwing a more descriptive error makes it look less like a node.js bug.